sessions/writing-guides.qmd

```diff
@@ -8,10 +8,20 @@
 
 <!-- TODO: Add an introduction here -->
 
+I tried to be as hierarchical as possible, starting from the main goal
+of the session and listing the concepts/skills needed to fulfill that
+objective. What do you think?
+
 **Session objectives**:
 
--   Create a README and vignette for package documentation
--   Ensure that documentation is consistent with up-to-date style
+-   MAIN: Learn how to create a README and vignette for package
+    documentation
```

Comment:
Here, the aim is to focus on the why as well as the what. For inspiration, see one session from the advanced course: https://r-cubed-advanced.rostools.org/sessions/pipelines#learning-objectives

In this case, it might look something like:

```suggestion
-   Learning outcome: Describe the different types of documentation and explain when and why to use them. Use this knowledge to create two types of documentation within the context of an R package.
```